.github/workflows/pypi-publish.yml

```diff
@@ -32,11 +28,13 @@ jobs:
     - name: Verify the distributions
       run: twine check dist/*
 
-    # Save the package to the GitHub Actions' artifact. It will be deleted after 90 days.
-    - name: Save dist directory
-      uses: actions/upload-artifact@v1
+    - name: Publish distribution to Test PyPI
+      # The following upload action cannot be executed in the forked repository.
+      if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags') && github.repository == 'optuna/optuna'
```

Comment:
Just wondering but we need the condition on `event_name` (or is it better to have in some sense)?

Reply:
I think it is no longer necessary because we remove `pull_request` from the trigger. I'll remove it. Thank you!

Reply:
I agree, thanks!